src/docs/arch-block.dox

```diff
@@ -1,9 +1,132 @@
 /*! @arch_page arch-block Block Manager
 
-The Block Manager manages the reading and writing of disk blocks
-in WiredTiger.  It does compression and encryption when these
-are configured.
+The WiredTiger block manager subsystem manages the reading and writing of data
+from the disk. It is designed to facilitate high performance, economic use of
+disk space and customisability.
+
+@section block What is a block?
+
+The block manager API introduces the concept of a "block" which represents a
+chunk of data that is stored on the disk and operated on as a single atomic
+unit. Each WiredTiger data file (any file in the home directory with the \c .wt
```

Comment:
"atomic" bothers me a little, I'm not sure why you're using that word, and it has other meanings around data structure that don't apply here?

Reply:
I'm using the word atomic because that means that it's irreducible which is what I'm trying to convey. I can remove it as "operated on as a single unit" is probably descriptive enough by itself.